AGENTS.md

```
@@ -2,6 +2,64 @@
+C_in  = (W_in * R_decode * H) / T_in
+```
+
+**Capability view (use when `T` is fixed):**
+
+```
+C_cap = Hmean(W_out * R_encode * H, W_in * R_decode * H) * C_ext^alpha * P_ij
+```
+
+**Monolithic alternative (rare):**
+
+```
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)